docs/integrations/sources/microsoft-lists.md

```diff
@@ -0,0 +1,33 @@
+# Microsoft Lists
+Microsoft Lists connector enables seamless data integration and synchronization between Microsoft Lists and other destination. The connector leverages Microsoft Graph API to retrieve list items efficiently, ensuring smooth workflows and real-time data accessibility
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `client_id_2` | `string` | Client ID.  |  |
+| `client_secret_2` | `string` | Client secret.  |  |
+| `application_id_uri` | `string` | Application Id URI.  |  |
+| `site_id` | `string` | Site Id.  |  |
+| `tenant_id` | `string` | Tenant Id.  |  |
+| `domain` | `string` | Domain.  |  |
```

Comment:
Provide more information about the parameters how to get them and configure them in a proper section in docs.

Reply:
ok